.circleci/config.yml

```diff
@@ -141,9 +141,14 @@ jobs:
       - run:
           name: wallet setup
           working_directory: packages/server-wallet
+          # Create databases for the server-wallet and interop-tests package tests
```

Comment:
These changes are part of a preceding [PR](https://github.com/statechannels/statechannels/pull/3422)

Reply:
Why are they still in this changeset?

Reply:
Not sure...

Reply:
I think it's because #3422 was merged with a squash commit, meaning Github does not recognize that the diff in a5c67e7 is already included in `master`.